osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/AdHocManagementServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package org.opensmartgridplatform.adapter.domain.core.application.services;
```

Comment:
The license header for new files should:
- refer to the current year when the file is added in the copyright line;
- mention Alliander N.V. instead of Smart Society Services B.V.